extensions.html

```diff
@@ -649,6 +649,168 @@ <h2>
         </p>
       </dd>
     </section>
+    <section>
+      <h2>
+        "extended" Gamepad Buttons Mapping
+      </h2>
+      <p>
+        This section defines extended gamepad buttons mapping beyonds the Standard Gamepads layout,
+        but are commonly found on some gamepad models .The <code>_ext</code> suffix is used to indicate
```

Comment:
The `_ext` suffix comes from OpenXR's input subpaths specification:

https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#_adding_input_sources_via_extensions

We should decide whether this attribute uses OpenXR input subpaths or something new that we want to define for ourselves. If we're using OpenXR subpaths then we should explicitly link to the OpenXR spec which explains how to extend it. If we're only using it for inspiration then I think we should drop the `_ext` suffix.